api/RTCSessionDescription.json

```diff
@@ -0,0 +1,151 @@
+{
+  "api": {
+    "RTCSessionDescription": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCSessionDescription",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Use with <code>-moz</code>"
```

Comment:
We have a special entry for prefixes. Do it that way:
"firefox": {
   "prefix": "-moz-",
   "version_added": "true"
}

(and no note)